pyphi/compute.py

```diff
@@ -229,11 +231,20 @@ def _evaluate_partition(uncut_subsystem, partition,
     log.debug("Evaluating partition {}...".format(partition))
     # Compute forward mip.
     forward_cut = Cut(partition[0], partition[1])
+    if not config.MATLAB_APPROX:
+        cut_mechanisms = utils.split_by_cut(uncut_subsystem, forward_cut)
```

Comment:
Let's call the config option `ASSUME_CUTS_CANNOT_CREATE_CONCEPTS` or something like that, since `MATLAB_APPROX` isn't very descriptive… then we can put a note in the docs further explaining it. In fact, maybe we should have a namespace in the config dictionary called `APPROXIMATIONS`, just to make it more explicit that this is not the actual value.